UDV-Core/examples/DocumentModel.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Would it be possible to regroup DocumentModel.json, optionsFilter.json, and schemaFilters.json within an examples/Document (or Contribute?) sub-directory to indicate they work together ?

Reply:
For the general demo (with all options / functionnalities), the files are in UDV-Core/examples/data
For the demo regarding consultDoc module, they are in: UDV-Core/src/Modules/ConsultDoc/examples